### ISSUE TITLE
 ocamlbuild.0.11.0 - Invalid field docdir

### ISSUE THREAD

**Issue by XVilka** [ocaml/ocamlbuild#241]
```
[WARNING] Errors in /home/user/.opam/4.04.1+fPIC/.opam-switch/build/ocamlbuild.0.11.0/ocamlbuild.install, some fields have been ignored:
            - At /home/user/.opam/4.04.1+fPIC/.opam-switch/build/ocamlbuild.0.11.0/ocamlbuild.install:34:0:
              Invalid field docdir
```

**Comment by gasche** [comment]
What version of OPAM are you using (`opam --version`)? It sounds rather old.

**Comment by XVilka** [comment]
@gasche it's opam-2.0.0_beta3

**Comment by gasche** [comment]
Ah indeed, it seems that opam 2.0 expects them [in the .install file](https://opam.ocaml.org/doc/2.0/Manual.html#ltpkgnamegtinstall), which means that we may need different logic for opam 1.x and 2.x.

**Comment by dbuenzli** [comment]
@gasche not sure what you are talking about but `docdir` isn't a valid field in v1.0 either. It's `doc`.

**Comment by gasche** [comment]
You... appear to be correct. I don't know why I used `docdir` in the first place.

**Comment by gasche** [comment]
I fixed this in #242, thanks!
